### PR TITLE
added flag for copy to clipboard after scan (-c)

### LINKED ISSUE #68: Automatically copy to clipboard
After scanning an OD, I need to press "C" to copy Reddit Markdown to clipboard.

Is there anyway to automate the process of "Copy to clipboard automatically"?

Needed for my python script :)

### PR DESCRIPTION
**Pull request by Chaphasilor**
- flag name is `-c` or `--clipboard`
- copies the reddit stats to the clipboard once the indexer is finished
- combined with `-q`, this closes #68

I also added some logic to detect if the clipboard copy worked and change the last `WriteLine()` accordingly.

Also: There are two failing test, but they were present right after forking, so I guess they aren't related...

Hope this is what you had in mind 😇

### CODE REVIEW

**Comment by Chaphasilor** [comment]
Whoops, forgot do edit the ReadMe 👀

BRB

File: OpenDirectoryDownloader/OpenDirectoryIndexer.cs [hunk 1]
```diff
@@ -480,14 +481,29 @@ public async void StartIndexingAsync()
                     Console.WriteLine("Finished indexing!");
 
                     Program.SetConsoleTitle($"✔ {Program.ConsoleTitle}");
-
+                    
+                    Boolean clipboardSuccess = false;
```
Comment (KoalaBear84) [inline OpenDirectoryDownloader/OpenDirectoryIndexer.cs:485]: Please change this tool "bool" and add a empty line after it. I thought it was possible to do single line changes in PR, can't find how 😮

**Review by KoalaBear84 — CHANGES_REQUESTED** [review]
Ohh, this is a different textbox :)

See comment :P

File: OpenDirectoryDownloader/OpenDirectoryIndexer.cs (hunk 1 above)
Comment (Chaphasilor) [reply OpenDirectoryDownloader/OpenDirectoryIndexer.cs:485]: will do!

**Review by KoalaBear84 — APPROVED** [review]

File: OpenDirectoryDownloader/OpenDirectoryIndexer.cs (hunk 1 above)
Comment (Chaphasilor) [reply OpenDirectoryDownloader/OpenDirectoryIndexer.cs]: I actually had a blank line in-between at first but though it'd be good to group things together xD

**Review by KoalaBear84 — APPROVED** [review]

File: OpenDirectoryDownloader/OpenDirectoryIndexer.cs (hunk 1 above)
Comment (KoalaBear84) [reply OpenDirectoryDownloader/OpenDirectoryIndexer.cs]: Haha. Yes, I understand. I like some space 🤣

**Comment by Chaphasilor** [comment]
🥳

**Comment by KoalaBear84** [comment]
👍👍👍

**Pull request merged**